### PR TITLE
Convert Ticket Assignments Manager to use Responsive Table; closes #2491.

### LINKED ISSUE #2491: Convert Ticket Assignments Manager to use Responsive Table
Currently TAM is using the AntD Table component which is not responsive nor a11y, so we need to convert it to use the EspressoTable component

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- Closes: https://github.com/eventespresso/event-espresso-core/issues/2491

### CODE REVIEW

**Comment by nicolad** [comment]
Had renamed `tableRows` to `bodyRows` in order to reflect a better relation with `headerRows` and `footerRows`.

**Comment by nicolad** [comment]
TAM on `EDTR/refactor-master` branch:

image

**Comment by nicolad** [comment]
TAM on `EDTR/master`:

image

File: assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx [hunk 1]
```diff
@@ -29,13 +29,13 @@ const TableView: React.FC<TableViewProps> = ({ className, displayStartOrEndDate,
 
 	return (
 		<ResponsiveTable
+			bodyRows={formRows}
 			className={{ tableClassName }}
 			headerRows={[headerRows]}
 			metaData={{
 				tableId: 'date-entities-list-view',
-				tableCaption: __('Event Dates', 'event_espresso'),
+				tableCaption: __('Event Datesaaas', 'event_espresso'),
```
Comment (tn3rb) [inline assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx]: 😲 ???

**Comment by nicolad** [comment]
TAM on current PR's branch:

image

**Comment by nicolad** [comment]
Also, take into account that as discussed with @manzoorwanijk there is more work that needs TBD to TAM.
This is related to disabled submit button when new ticket is created and not only.

File: assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx (hunk 1 above)
Comment (nicolad) [reply assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx]: oops

Comment (nicolad) [reply assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx]: no, Bogdan was sleeping when the change has been made

File: assets/src/domain/eventEditor/ui/ticketAssignmentsManager/table/getHeaderRows.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		type: 'cell',
+		className: 'ee-rspnsv-table-column-huge',
+		value: __(ticket.name, 'event_espresso'),
+	}));
+
+	return [
+		{
+			cells: [emptyCell, ...cells],
+			className: 'ticket-assignment-manager-table-header-row',
+			key: 'ticket-assignment-manager-table-header-row',
```
Comment (manzoorwanijk) [inline assets/src/domain/eventEditor/ui/ticketAssignmentsManager/table/getHeaderRows.tsx]: This key should be unique for each row

File: assets/src/domain/eventEditor/ui/ticketAssignmentsManager/table/getBodyRows.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+				type: 'cell',
+				className,
+				value: <BodyCell datetime={datetime} ticket={ticket} />,
+			};
+		});
+
+		return {
+			cells: [datetimeCell, ...cells],
+			className: 'ticket-assignment-manager-table-body-row',
+			key: 'ticket-assignment-manager-table-body-row',
```
Comment (manzoorwanijk) [inline assets/src/domain/eventEditor/ui/ticketAssignmentsManager/table/getBodyRows.tsx]: This key should be unique for each row

**Pull request merged**